crates/oxc_ast_macros/src/lib.rs

```diff
@@ -16,9 +25,23 @@ use std::str::FromStr;
 #[proc_macro_attribute]
 #[allow(clippy::missing_panics_doc)]
 pub fn ast(_args: TokenStream, input: TokenStream) -> TokenStream {

```

Comment:
Doc comment above this function needs updating to reflect that this is a real macro now and adds `#[repr]` attribute to types.